common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -3194,6 +3194,15 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Age of table/partition's oldest aborted transaction when compaction will be triggered. " +
         "Default time unit is: hours. Set to a negative number to disable."),
 
+    HIVE_COMPACTOR_ACTIVE_DELTA_DIR_THRESHOLD("hive.compactor.active.delta.dir.threshold", 200,
+        "Number if active delta directories under a given table/partition."),
```

Comment:
I think the descriptions here should reflect that these are thresholds, and that logging will happen if they are passed